test/hotspot/jtreg/vmTestbase/nsk/jdwp/ThreadReference/Interrupt/interrupt001.java

```diff
@@ -90,7 +90,10 @@ public class interrupt001 {
      * Start test from command line.
      */
     public static void main (String argv[]) {
-        System.exit(run(argv,System.out) + JCK_STATUS_BASE);
+        int result = run(argv,System.out);
```

Comment:
```suggestion
        int result = run(argv, System.out);
```

Reply:
Yes, this missing space is all over this fix. Need to fix this formatting.